src/main/java/org/quantumbadger/redreader/activities/PostSubmitActivity.java

```diff
@@ -187,10 +188,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		if (item.getItemId() == android.R.id.home) {
 			finish();
 		} else if(item.getTitle().equals(getString(R.string.comment_reply_send))) {
-			final Intent captchaIntent = new Intent(this, CaptchaActivity.class);
-			captchaIntent.putExtra("username", (String)usernameSpinner.getSelectedItem());
-			startActivityForResult(captchaIntent, REQUEST_CAPTCHA);
-
+			String subreddit = subredditEdit.getText().toString();
+			String postTitle = titleEdit.getText().toString();
+			String text = textEdit.getText().toString();
+			if (subreddit.isEmpty()) {
+				Toast.makeText(this, "Specify subreddit", Toast.LENGTH_SHORT).show();
```

Comment:
This string (and the ones on line 198 and 201) should be declared at the end of strings.xml (to allow for translation).